plutus-playground-server/test/Playground/UsecasesSpec.hs

```diff
@@ -142,7 +142,7 @@ vestingTest =
     vestFundsEval = mkEvaluation [vestFunds w2]
     vestAndPartialRetrieveEval =
         mkEvaluation
-            [vestFunds w2, AddBlocks 10, retrieveFunds w1 5, AddBlocks 5]
+            [vestFunds w2, AddBlocks 20, retrieveFunds w1 5, AddBlocks 1]
```

Comment:
I'm not sure why this test ever passed - the contract releases the first tranche at slot no. 20!